command/operator_diagnose.go

```diff
@@ -373,6 +373,57 @@ func (c *OperatorDiagnoseCommand) offlineDiagnostics(ctx context.Context) error
 		return nil
 	})
 
+	diagnose.Test(ctx, "seal-transit-tls-checks", func(ctx context.Context) error {
+		var checkSealTransit bool
+		for _, seal := range config.Seals {
+			if seal.Type == "transit" {
+				checkSealTransit = true
+
+				tlsSkipVerify, ok := seal.Config["tls_skip_verify"]
+				if !ok {
+					// Just warn here and not returning an error
+					diagnose.Warn(ctx, "Missing tls_skip_verify in the config")
```

Comment:
I have not returned anything here. This has made the output of diagnose to misbehave. In particular, "warn" message has indented a little and new empty line was printed above it. 
here is a snapshot:

<img width="1078" alt="Screen Shot 2021-06-23 at 4 54 53 PM" src="https://user-images.githubusercontent.com/83242695/123166781-c94b8880-d443-11eb-9336-357328f3b8dd.png">

Reply:
Also, it seems that the name of the diagnose test is also not printed, which is seal-transit-tls-checks.
Here is a snapshot when I do return nil right after that warning
<img width="1093" alt="Screen Shot 2021-06-23 at 4 57 54 PM" src="https://user-images.githubusercontent.com/83242695/123167147-3101d380-d444-11eb-8e6d-53aed5c2c500.png">

Reply:
Hopefully this is fixed after merging main?